JWBestPracticeApps/FeedTableViewController/FeedItemCell.swift

```diff
@@ -15,6 +15,25 @@ class FeedItemCell: UITableViewCell {
     
     @IBOutlet weak var containerView: UIView!
     
+    var player: JWPlayerController? {
+        didSet {
+            guard let playerView = player?.view else {
+                return
+            }
+            
+            containerView.addSubview(playerView)
+            playerView.constraintToSuperview()
+        }
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        
+        for view in containerView.subviews {
+            view.removeFromSuperview()
+        }
```

Comment:
I have just one suggestion. Apple's suggests do not modify the content in the `prepareForReuse` method.

> For performance reasons, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state. The table view's delegate in tableView:cellForRowAtIndexPath: should always reset all content when reusing a cell.
https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse?language=objc

Reply:
I have fixed this by moving the removal code into **willSet** for the player property.

Reply:
That make sense to me.